README.md

```diff
@@ -1,2 +1,120 @@
-# mailgun-relayery
-A Go server providing an additional security layer for using the MailGun API
+Mailgun Relayery
+================
+
+Mailgun Relayery is a tool providing an additional security layer for the MailGun API.
+
+**The problem**: MailGun gives out a single API key to each account. For entities dealing with multiple clients or 
+applications, giving out their API key to each of them constitutes a blatant security violation.
+
+**The solution**: _Mailgun Relayery_ acts as a middle layer between message senders and the MailGun API.
+Senders are issued descriptor strings (acting like usernames) and HTTP authentication tokens, which 
```

Comment:
s/HTTP//

There is no such a term as "HTTP token" -- it's just an authentication token (_i.e._ it's the API/logical layer, not HTTP/protocol layer).